cdap-security/src/test/java/co/cask/cdap/security/authorization/InMemoryAuthorizer.java

```diff
@@ -168,24 +170,32 @@ public void removeRoleFromPrincipal(Role role, Principal principal) throws RoleN
         privileges.addAll(getPrivileges(role));
```

Comment:
Similarly in the line above, better not to call `listRoles` but rather loop over the `roleToPrincipals` directly.